server/api/api.go

```diff
@@ -797,8 +861,19 @@ func (a *API) handleGetSharing(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	a.logger.Debug("GET sharing", mlog.String("rootID", rootID))
 	jsonBytesResponse(w, http.StatusOK, sharingData)
+
+	if sharing == nil {
+		sharing = &model.Sharing{}
+	}
+	a.logger.Debug("GET sharing",
```

Comment:
Instead of this, would setting the value as `N/A` be better? It will be clearer in the logs IMO.

Reply:
I think the issue here is that this API should not allow empty responses.  IMO a 404 should be returned if no sharing record exists to be consistent with other Rest APIs.  I'm not sure if the client can handle that.

Reply:
Agree on returning 404. I can create a card for it. Can we update the audit code here assuming we're returning a 404? Refactoring to 404 and updating audit aren't blocked on each other IMO, unless they are?